pkg/s3select/sql/value.go

```diff
@@ -66,6 +70,15 @@ func (value *Value) MarshalJSON() ([]byte, error) {
 	return json.Marshal(value.value)
 }
 
+// NullValue - returns underlying null value. It panics if value is not null type.
+func (value *Value) NullValue() *struct{} {
+	if value.valueType == Null {
+		return nil
+	}
+
+	panic(fmt.Sprintf("requested bool value but found %T type", value.value))
```

Comment:
Is there  typo, should this be
`panic(fmt.Sprintf("requested null value but found %T type", value.value))`